pkg/volume/util/nestedpendingoperations/nestedpendingoperations.go

```diff
@@ -92,6 +92,7 @@ func (grm *nestedPendingOperations) Run(
 	grm.lock.Lock()
 	defer grm.lock.Unlock()
 
+	// search for an existing operation
```

Comment:
@pmorie @saad-ali This is nice to have, but should we really bundle this in this PR? Is it unrelated? Same for the following comment.